### PR TITLE
Avoid division by zero

### PR DESCRIPTION
**Pull request by bajb**

### CODE REVIEW

File: src/NXP/Classes/Token/TokenDivision.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function execute(&$stack)
     {
         $op2 = array_pop($stack);
         $op1 = array_pop($stack);
-        $result = $op1->getValue() / $op2->getValue();
+        $result = $op2->getValue() > 0 ? $op1->getValue() / $op2->getValue() : 0;
```
Comment (NeonXP) [inline src/NXP/Classes/Token/TokenDivision.php]: Hello! Thank for your work! What about negative second operator? And why zero result, mb infinity?

**Comment by NeonXP** [comment]
Hello! Thank for your work! What about negative second operator? And why zero result, mb infinity?

**Comment by bajb** [comment]
I think in this context, the best value to work with would be 0, and would be expected in many cases.  INF I would expect to add more confusion.

**Comment by NeonXP** [comment]
Thanks!

**Comment by NeonXP** [comment]
Added test for division by zero

**Pull request merged**